configuration/configuration.py

```diff
@@ -56,23 +56,15 @@ def read_secret(secret_name):
         'PORT': int(os.environ.get('REDIS_PORT', 6379)),
         'PASSWORD': os.environ.get('REDIS_PASSWORD', read_secret('redis_password')),
         'DATABASE': int(os.environ.get('REDIS_DATABASE', 0)),
-        'DEFAULT_TIMEOUT': int(os.environ.get('REDIS_TIMEOUT', 300)),
-        'SSL': os.environ.get('REDIS_SSL', 'False').lower() == 'true',
-    },
-    'webhooks': { # legacy setting, can be removed after Netbox seizes support for it
-        'HOST': os.environ.get('REDIS_HOST', 'localhost'),
-        'PORT': int(os.environ.get('REDIS_PORT', 6379)),
-        'PASSWORD': os.environ.get('REDIS_PASSWORD', read_secret('redis_password')),
-        'DATABASE': int(os.environ.get('REDIS_DATABASE', 0)),
-        'DEFAULT_TIMEOUT': int(os.environ.get('REDIS_TIMEOUT', 300)),
+        'RQ_DEFAULT_TIMEOUT': int(os.environ.get('REDIS_TIMEOUT', 300)),
```

Comment:
The RQ_DEFAULT_TIMEOUT seems to be a standalone variable in the original configuration example and enclosed in the 'Redis' subsection.
See [here](https://github.com/netbox-community/netbox/blob/e304f105c99e25a89c9f6714e36cb095400ec521/netbox/netbox/configuration.example.py#L233)